challenges/kali/challenges/hashcat/template.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: ctf.acasi.info/v1alpha1
+kind: EnvTemplate
+metadata:
+  name: kali-hashcat
+spec:
+  pods:
+    - metadata:
+        name: penimage
+        labels:
+          app: penimage
+      spec:
+        containers:
+          - name: penimage
+            image: ghcr.io/acasi-ctf/challenges/kali/penimage:latest
```

Comment:
This image doesn't exist.

Reply:
Oh stupid, I put that as a placeholder so I wouldn't forget.... haha and here we are